### ISSUE TITLE
Forgematica and MaFgLib

### ISSUE THREAD

**Issue by tombaku** [ThinkingStudios/Litematica-Forge#55]
After adding both Forgematica 0.1.1-mc1.20.4 and MaFgLib 0.1.2-mc1.20.4 to a 1.20.4 instance with Neoforge 20.4.214 I encountered an error where Forgematica wouldn't recognise the install of MaFgLib and would error on startup with

```
	Mod File: <install directory>/mods/Forgematica-0.1.1-mc1.20.4.jar
	Failure message: Mod forgematica requires mafglib 0.1.4 or above
		Currently, mafglib is 0.1.2-mc1.20.4
```
I think when updating to 1.20.4 the version number was reset and not accounted for somewhere
I'm not sure where this should be fixed either in Forgematica or MaFgLib's code but it should be quick at least

**Comment by TexBlock** [comment]
Forgematica issue

**Comment by TexBlock** [comment]
fixed in `0.1.2`
